Resources/Maps/_NF/POI/medical.yml

```
@@ -2035,6 +2022,18 @@ entities:
       rot: -1.5707963267948966 rad
       pos: 8.5,-26.5
       parent: 1
+- proto: AirlockFrontierBridgeLocked
```

```suggestion
- proto: AirlockFrontierCommandLocked
```
Bridge is for STC/SR access, Command is for the SR's office.  This'll work.